### ISSUE TITLE
Import alias files containing references between aliases

### ISSUE THREAD

**Issue by FrankWeis** [opnsense/core#3522]
**Important notices**
Before you add a new report, we ask you kindly to acknowledge the following:

[-] I have read the contributing guide lines at https://github.com/opnsense/core/blob/master/CONTRIBUTING.md

[-] I have searched the existing issues and I'm convinced that mine is new.

**Describe the bug**

**To Reproduce**

**Expected behavior**

**Screenshots**

**Relevant log files**
If applicable, information from log files supporting your claim.

**Additional context**

**Environment**
Software version used and hardware type if relevant.
e.g.:

OPNsense 19.1.1 (amd64, OpenSSL).
Intel® Xeon™ E3-1225V5 3.3Ghz Quad Core
Network Intel® I210-AT

**Comment by FrankWeis** [comment]
I believe I have found a bug with export/import, when aliases reference other aliases, the order is important.

I had sth like:
`
voipservers={voip1,voip2}

voip1={...}

voip2={...}
`
in a file, and import choked on voip2 is not an IP address nor a hostname.

I then deleted the definition of the voipservers from the file and it imported  fine.
I then re-imported the original file (so voip1 and voip2 where already known) and it worked fine too.

I tried fixing the order in the file (copy the definition of voipservers after voip1 and voip2), but that didn't help.

Apart from doing multiple passes on import, I don't see an easy way to fix this.
